src/HIE/Bios/Cradle.hs

```diff
@@ -326,6 +340,7 @@ biosCradle wdir biosCall biosDepsCall =
     , cradleOptsProg   = CradleAction
         { actionName = Types.Bios
         , runCradle = biosAction wdir biosCall biosDepsCall
+        , getGhcPath = findExecutable "ghc"
```

Comment:
Afaik the dependencies of the bios cradle is a responsability of the bios program (see biosDeps): should not be the ghc used by that bios program returned by itself? Or at least being able to take in account it and fallback to the ghc on path.

Reply:
Is there a way for a bios program to return the path to the GHC it intends to use? From my understanding it just returns newline separated arguments in stdout

Reply:
It may still want define the libdir, right?

Reply:
I'm not sure if there will be a good way to have the bios program pass the library directory back, seeing as programs are only really aware of filepaths being passed to them, either through the shell arguments or by the `HIE_BIOS_ARG` environment variable. Also I can't really imagine how the bios programs are going to return the library directory in the first place. 
With that said, the current way haskell-language-server works out the project version is by calling `ghc` on the path anyway, and this should fall back to the hardcoded ghc-paths should that fail. So I don't think there would be much of a difference.

Reply:
Well, an option could be add another optional config option like the existing one for dependencies:
```
cradle:
  bios:
    program: ./flags.sh
    dependency-program: ./dependency.sh
    ghc-path-program: ./ghc-path.sh
```
But i am not aware of a bios program that needs it so i would merge this without it and add it on concrete user request afterwards